guides/source/active_model_basics.md

```diff
@@ -550,40 +1316,40 @@ irb> person.valid?
 => false
 
 # When the confirmation doesn't match the password.
-irb> person.password = 'aditya'
-irb> person.password_confirmation = 'nomatch'
+irb> person.password = "aditya"
+irb> person.password_confirmation = "nomatch"
 irb> person.valid?
 => false
 
 # When the length of password exceeds 72.
-irb> person.password = person.password_confirmation = 'a' * 100
+irb> person.password = person.password_confirmation = "a" * 100
 irb> person.valid?
 => false
 
 # When only password is supplied with no password_confirmation.
-irb> person.password = 'aditya'
+irb> person.password = "aditya"
 irb> person.valid?
 => true
 
 # When all validations are passed.
-irb> person.password = person.password_confirmation = 'aditya'
+irb> person.password = person.password_confirmation = "aditya"
 irb> person.valid?
 => true
 
 irb> person.recovery_password = "42password"
 
-irb> person.authenticate('aditya')
+irb> person.authenticate("aditya")
 => #<Person> # == person
-irb> person.authenticate('notright')
+irb> person.authenticate("notright")
 => false
-irb> person.authenticate_password('aditya')
+irb> person.authenticate_password("aditya")
 => #<Person> # == person
-irb> person.authenticate_password('notright')
+irb> person.authenticate_password("notright")
 => false
 
-irb> person.authenticate_recovery_password('42password')
+irb> person.authenticate_recovery_password("42password")
```

Comment:
Maybe illustrate that 'password' and 'recovery_password' are different with this added example:

```suggestion
irb> person.authenticate_recovery_password("aditya")
=> false
irb> person.authenticate_recovery_password("42password")
```